xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodec.cpp

```diff
@@ -16,9 +16,6 @@
 //******************************************************************************
 // VideoPicture
 //******************************************************************************
-
-VideoPicture::VideoPicture() = default;
-
```

Comment:
Sometimes having this in the cpp is the better approach. Namely, if the class has members which are forward declared, then this forward declare has to be replaced with an include in the class' header file. Otherwise the compiler does not know how to compile the function it generates. Just as a note, seems to work just fine here without need to include other headers.